README.md

```diff
@@ -47,79 +47,143 @@ Several features are in beta testing and will be available in the next release s
 
 *If your want to join in our development team, please contact huaizhen001 @ e.ntu.edu.sg*
 
-## Installation
+## Demo
 
-### using pip
+The below figures illusrates the 
+| Web frontend |   Workflow     |
+|:------------:|:--------------:|
+| <img src="https://i.loli.net/2020/12/10/4FsfciXjtPO12BQ.png" alt="drawing" width="500"/> | <img src="https://i.loli.net/2020/12/10/8IaeW9mS2NjQEYB.png" alt="drawing" width="500"/>    |
+
+
+## Installation Guide
+
+### Prerequisites
+
+- A GNU/Linux environment(Ubuntu preferred)
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/) (for Installation via Docker)
+- [TVM](https://github.com/apache/incubator-tvm) and `tvm` Python module
+- [TensorRT](https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html) and Python API
+- Python >= 3.7
+
+### Installation using  pip
 
 ```shell script
-# upgrade installation packages first
-pip install -U setuptools requests
 # install modelci from GitHub
 pip install git+https://github.com/cap-ntu/ML-Model-CI.git@master
 ```
 
-### create conda workspace 
+Once you have installed, make sure the docker daemon is running, then you can start MLModelCI service on a leader server by:
 
-**Note**
-- Conda and Docker are required to run this installation script.
-- To use TensorRT, you have to manually install TensorRT (`sudo` is required). See instruction 
-[here](https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html). 
+```bash
+modelci service init
+```
 
-```shell script
-bash scripts/install.sh
+![CLI start service](https://i.loli.net/2021/04/12/wtyMuIlaJdsVchX.gif)
+
+Or stop the service by:
+
+```bash
+modelci service stop
 ```
 
-### Docker
+![CLI stop service](https://i.loli.net/2021/04/12/Hdi4JTclKkEaLfh.gif)
```

Comment:
Why your characters have space in between? Is this a font encoding problem?

Reply:
It's related to the font, I'll use monospaced font and replace all the images.